obelisk_ws/src/obelisk_ros/config/dummy_py.yaml

```diff
@@ -49,11 +49,17 @@ onboard:
   # sensing:
```

Comment:
please revert these changes and make a new yaml file specifically for the leap hand with hardware (and probably make another one for sim)